vnext/ReactWindowsCore/WinRTWebSocketResource.cpp

```diff
@@ -4,13 +4,16 @@
 // clang-format off
 #include "WinRTWebSocketResource.h"
 
+#include <ReactUWP/Utils/CppWinrtLessExceptions.h>
 #include <Unicode.h>
 #include <Utilities.h>
 
 // Windows API
 #include <winrt/Windows.Foundation.Collections.h>
 #include <winrt/Windows.Security.Cryptography.h>
+#include <atlbase.h>
```

Comment:
we shouldn't need atlbase? what's requiring it?

Reply:
Resolved.

Using `winrt::com_ptr` instead of `CComPtr`.